apps/ui/src/DesktopApp/home/home.tsx

```diff
@@ -0,0 +1,213 @@
+        <ToggleButtonGroup
+          exclusive
+          value={role}
+          size={'small'}
+          color={'primary'}
+          onChange={handleChangeRole}
+        >
+          <ToggleButton value={'driver'}>Driver</ToggleButton>
+          <ToggleButton value={'rider'}>rider</ToggleButton>
+        </ToggleButtonGroup>
+        <Box
+          sx={{
+            height: '64%',
+            display: 'flex',
+            flexDirection: 'column',
+            justifyContent: 'space-between',
+          }}
+        >
+          <List sx={{ mt: '10%' }}>
+            {Object.keys(menuOptions).map((text) => (
+              <Box key={text}>
+                <ListItem
+                  sx={{
+                    paddingRight: '0',
+                    color: defaultTheme.palette.info.dark,
+                    '&:hover': {
+                      backgroundColor: defaultTheme.palette.action.hover,
+                    },
+                  }}
+                >
+                  <ListItemButton
+                    onClick={() =>
+                      setOpenedPage(text as keyof typeof menuOptions)
+                    }
+                  >
+                    <ListItemText primary={text} />
+                    <ChevronRightIcon color={'action'} />
+                  </ListItemButton>
+                </ListItem>
+                <Divider />
+              </Box>
+            ))}
+          </List>
+          <Box>
+            <Box sx={{ display: 'flex', margin: '4%', alignItems: 'center' }}>
+              <Avatar
+                src={user?.avatarUrl}
+                sx={{ height: '35px', width: '35px' }}
+              />
+              <Typography sx={{ marginLeft: '5%', marginTop: '2%' }}>
+                Hi {user?.firstName} {user?.lastName[0]}. 👋
+              </Typography>
+            </Box>
+
+            <Divider />
+            <ListItem
+              sx={{
+                paddingRight: '0',
+                color: defaultTheme.palette.info.dark,
+                '&:hover': {
+                  backgroundColor: defaultTheme.palette.action.hover,
+                },
+              }}
+            >
+              <ListItemButton onClick={() => signOut()}>
+                <ListItemText primary={'Log out'} />
+                <ChevronRightIcon color={'action'} />
+              </ListItemButton>
+            </ListItem>
+          </Box>
+        </Box>
+      </Box>
+      {menuOptions[openedPage]}
+      <Map
+        shouldShowSearch={false}
+        focusLocation={selectedRide?.startLocation}
+        markers={
+          flatten(
+            communities.map((community) =>
+              community.rides.map((ride: Ride) => ({
+                geocode: ride.startLocation,
+                popUp: `${ride.driver.firstName} ${ride.driver.lastName} going to ${ride.destinationName}`,
+              })),
+            ),
+          ) as MarkerInfo[]
+        }
+      />
+    </Page>
+  );
+};
+
+export { Home };
```

Comment:
Feels that there are double things here that can be combined with the display of the mobile